build-and-benchmark-py-project.sh

```diff
@@ -58,6 +59,16 @@ if [[ "$output" = *"worse"* ]]; then
 fi
 
 if [[ "$CI_PROJECT_NAMESPACE" == "inducer" ]]; then
+  cd benchmark-data
+  git add $PROJECT
+  export GIT_AUTHOR_NAME="Automated Benchmark Bot"
+  export GIT_AUTHOR_EMAIL="bot@gitlab.tiker.net"
+  export GIT_COMMITTER_NAME=$GIT_AUTHOR_NAME
+  export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
+  git commit -m "Update benchmark data for $main_commit" --allow-empty
+  ssh-agent bash -c "ssh-add $PWD/../.deploy_key; git push"
+  cd ..
   git branch -v
+  mkdir -p ~/.scicomp-benchmarks/asv
   asv publish --html-dir ~/.scicomp-benchmarks/asv/$PROJECT
```

Comment:
I'm not a huge fan of the fact that this is not local to the build. Any chance we could move this?

Reply:
Fixed it